axonserver/src/main/java/io/axoniq/axonserver/localstorage/file/SegmentBasedEventStore.java

```diff
@@ -147,7 +130,6 @@ public void processEventsPerAggregate(String aggregateId, long firstSequenceNumb
                                                                                       eventConsumer,
                                                                                       Long.MAX_VALUE,
                                                                                       minToken));
-        aggregateReadTimer.record(System.currentTimeMillis() - before, TimeUnit.MILLISECONDS);
```

Comment:
I know that customers are monitoring the aggregateReadTimer, if we no longer record the values this must be documented for users.

Reply:
aggregateReadTimer is a duplicate of event_stream metrics, and event_stream metric is more granular, it records duration on different layers. I agree that we should document it